automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/workspace/WorkspaceWriterSpec.scala

```diff
@@ -26,7 +27,7 @@ class WorkspaceWriterSpec extends FreeSpec with Matchers with WebBrowserSpec wit
 
   "A user" - {
     "who has writer access" - {
-      "should only see the estimated monthly storage fee in the Project Cost section of the summary page" in {
+      "should only see the estimated monthly storage fee in the Project Cost section of the summary page" taggedAs Tags.SmokeTest in {
```

Comment:
is this really worthwhile as a UI smoketest?

At the API level, we can test whether or not the APIs return costs and billing links, etc. That just leaves this test asserting that the UI properly displays what is returned from the APIs. That's valid, but is it worth it?

Reply:
you're right about the reason. I picked this one to assert UI can display cost information. This is a relatively new UI feature. I think user probably cares about the cost, so I think it was something that should continue to be tested. Tests are run in parallel, wouldn't affect playback time much at all.